dp/cloud/python/magma/radio_controller/tests/test_utils/active_mode_config_builder.py

```diff
@@ -3,10 +3,10 @@
 from typing import Optional
 
 from dp.protos.active_mode_pb2 import (
-    ActiveModeConfig,
```

Comment:
since ActiveModeConfig is out of the picture, maybe let's rename the class file altogether or move this code to db_cbsd_builder.py?
Also changing the builder class name seems reasonable

Reply:
Changed to `ActiveModeCbsdBuilder` (we already have `DbCbsdBuilder`, so just `CbsdBuilder` wasn't enough)